cmake/projects/restinio/ci/matrix.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
I don't know if this file is needed. I thought there is a default `matrix.json` and this override is only needed if you need to change toolchains

Reply:
restinio is not compatible with VS2015, hence me commenting it out. If there is a better way to do this, I'd be glad to hear.